instrumentation/servlet/servlet-3.0/javaagent/src/test/java/RequestDispatcherServlet.java

```diff
@@ -37,6 +41,12 @@ protected void service(HttpServletRequest req, HttpServletResponse resp)
       String target = req.getServletPath().replace("/dispatch", "");
       ServletContext context = getServletContext();
       RequestDispatcher dispatcher = context.getRequestDispatcher(target);
+      // for HTML test case, set content type before calling include as
+      // "include" reject modify on resp later
+      // check https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html
```

Comment:
```suggestion
      // for HTML test case, set the content type before calling include because
      // setContentType will be rejected if called inside of include
```